blocks/header/header.css

```diff
@@ -30,15 +23,62 @@ header nav[aria-expanded="true"] {
   min-height: 100vh;
 }
 
-@media (min-width: 600px) {
-  header nav {
-    padding: 0 2rem;
+
+
+@media (min-width: 898px) and (max-width: 1300px) {
+  header .nav-bottom {
+      margin-top: 0.75rem;
+      margin-bottom: 0.75rem;
   }
 }
 
+header .nav-top {
+  display: block;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  --bg-opacity: 1;
+  background: rgba(231,233,234,var(--bg-opacity));
+  position: relative;
+  box-sizing: border-box;
+  -webkit-tap-highlight-color: rgba(0,0,0,0);
+  cursor: pointer;
+  float: right;
+  width: 100%;
+}
+
+.nav-top>.cmp-container {
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  flex-direction: row;
+  margin-left: auto!important;
+  margin-right: auto!important;
+  max-width: 73.123rem!important;
+  float: none!important;
+  -webkit-box-pack: end;
+  justify-content: flex-end;
+}
+
+header .nav-bottom {
+  margin-top: 0.75rem;
+  margin-bottom: 0.75rem;
+  clear: both!important;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 73.123rem;
+  float: right;
+  min-height: 75px;
+}
+
 @media (min-width: 900px) {
```

Comment:
I see four different behavior in the header for the following sizes:
- mobile
- tablet
- small desktop
- large desktop

those behaviors should be replicated in franklin for all the sizes